src/main/java/com/github/mauricioaniche/ck/util/JDTUtils.java

```diff
@@ -140,4 +140,12 @@ private static String formatSignature(List<String> parameters){
 				(parameterCount > 0 ? "]" : "")
 		);
 	}
+
+	//get the simple name from the fragments of a variable or field declaration, e.g. [a=10] -> a
+	//Be aware: the function might return the empty string
+	public static String getVariableName(List<VariableDeclarationFragment> fragments){
+		if(fragments.size() > 0)
+			return fragments.get(0).getName().toString();
```

Comment:
Why do you get only the first one? I suppose a code like this would break your code:

```
class X {
  private int a = 0, b=20;
}
```

Could you write a test for that?

Reply:
In this case, this method would return a `List<String>` with all the variables

Reply:
I wrote a test for this and fixed the issue.